etc/scripts/julia_wrapper.jl

```diff
@@ -84,17 +84,17 @@ open(output_path, "w") do io
     for (me_fun, me_types, me) in m_methods
         io_buf = IOBuffer() # string buffer
         if format == "typed"
-            ir, retval = InteractiveUtils.code_typed(me_fun, me_types, debuginfo=debuginfo)[1]
+            ir, retval = InteractiveUtils.code_typed(me_fun, me_types; optimize, debuginfo)[1]
             Base.IRShow.show_ir(io_buf, ir)
         elseif format == "lowered"
-            cl = Base.code_lowered(me_fun, me_types, debuginfo=debuginfo)
+            cl = Base.code_lowered(me_fun, me_types,; debuginfo)
```

Comment:
Pardon me as I don't know Julia (I've been meaning to learn it), is `,;` correct syntax here?

Reply:
It's fine, but cleaner would have been the single `;`. I.e. `,` is a trailing comma.

```
julia> f(a,; kwargs...) = a, kwargs
f (generic function with 1 method)

julia> f(1; b=2)
(1, Base.Pairs(:b => 2))
```

and `;` denotes the beginning of keyword-arguments.